### PR TITLE
Add missing GoogleService-Info.plist for both objc and swift apps

### PR DESCRIPTION
**Pull request by renkelvin**

### CODE REVIEW

**Comment by ulukaya** [comment]
Please do not merge!

**Comment by renkelvin** [comment]
Not sure why it works locally but breaks on Travis.
What I want to fix is to remove unused `AuthenticationExampleSwift/AuthenticationExample.entitlements` and add `GoogleService-Info.plist` to xcodeproj.

**Comment by morganchen12** [comment]
GoogleService-Info.plist files are ignored by git according to our [gitignore rules](https://github.com/firebase/quickstart-ios/blob/master/.gitignore#L5).

File: authentication/AuthenticationExample.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -15,6 +15,8 @@
 		10B053F91C6631CD0061077D /* UIViewController+Alerts.m in Sources */ = {isa = PBXBuildFile; fileRef = 10B053F81C6631CD0061077D /* UIViewController+Alerts.m */; };
 		10B053FB1C6945D80061077D /* UIViewController+Alerts.m in Sources */ = {isa = PBXBuildFile; fileRef = 10B053F81C6631CD0061077D /* UIViewController+Alerts.m */; };
 		10B054031C6A3C630061077D /* Main.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 10B054011C6A3C630061077D /* Main.storyboard */; };
+		403580D922A9C09400F6B217 /* GoogleService-Info.plist in Resources */ = {isa = PBXBuildFile; fileRef = 403580D822A9C09400F6B217 /* GoogleService-Info.plist */; };
+		403580DA22A9C0BE00F6B217 /* GoogleService-Info.plist in Resources */ = {isa = PBXBuildFile; fileRef = 403580D822A9C09400F6B217 /* GoogleService-Info.plist */; };
```
Comment (ulukaya) [inline authentication/AuthenticationExample.xcodeproj/project.pbxproj]: please delete the references to the .plist file file.

**Comment by ulukaya** [comment]
.plist file is added by developers per instructions and not stored in this project. Let's cancel this PR. Do you mind sending a new one just for the deletion of entitlement file. Or I can update it quick whichever you prefer.

**Pull request closed without merging**